app-misc/lcd4linux/lcd4linux-0.11.0_pre20170527-r2.ebuild

```diff
@@ -0,0 +1,186 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools flag-o-matic python-single-r1
+
+DESCRIPTION="A small program that grabs information and displays it on an external LCD"
+HOMEPAGE="https://lcd4linux.bulix.org/"
+SRC_URI="https://www.bl4ckb0x.de/files/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dmalloc outb"
+REQUIRED_USE="
+	?? ( lcd_devices_hd44780 lcd_devices_hd44780-i2c )
+	python? ( ${PYTHON_REQUIRED_USE} )
```

Comment:
this is odd; `python` use flag is defined way below. none of the other flags have been defined at that point either. please, merge this with the second `REQUIRED_USE` block in line 51.

Reply:
I disagree. The location in line 51 explictly adds the lcd_driver stuff. Why should I add there my python use flag? Since it's stand alone, It's easier to see in the beginned as somewhere in the middle?